hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -8702,8 +8703,13 @@ private void interruptRegionOperations() {
     for (Map.Entry<Thread, Boolean> entry : regionLockHolders.entrySet()) {
       // An entry in this map will have a boolean value indicating if it is currently
       // eligible for interrupt; if so, we should interrupt it.
+      Thread key = entry.getKey();
       if (entry.getValue().booleanValue()) {
-        entry.getKey().interrupt();
```

Comment:
How about we keep this change as is, and log whether the thread is interruptible as part of the common log above:
```
          // print all thread stacks which are still holding locks and are the cause of RS abort
          for (Map.Entry<Thread, Boolean> rslocks : regionLockHolders.entrySet()) {
            LOG.warn("Thread still holding lock: {} , interruptible: {} , Stack trace: {}",
              rslocks.getKey(), rslocks.getValue(), Threads.printStackTrace(rslocks.getKey()));
          }
```

This will help make logs appear at common place and make it more easier to debug.